Firebase/Core/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v6.3.3 -- M59
+- [changed] Carthage and zip file distributions are now built with Xcode 11.
```

Comment:
nit: Maybe we should be specific and specify Xcode 11.0 version just to clear up any confusion? @paulb777 WDYT?

Reply:
Definitely we should exactly specify the Xcode version.

Reply:
Done (internal CL updated as well).